<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>127</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>378</width>
    <height>127</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HT tracker settings</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical camera field of view</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="cameraFOV">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="minimum">
    <double>35.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="value">
    <double>69.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames per second</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraFPS">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">Default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera index</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cameraIndex">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Enable axes</string>
   </property>
   <widget class="QCheckBox" name="rx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RZ</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ty">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tz">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TZ</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
